### DISCUSSION TITLE
Configuration Options

### DISCUSSION THREAD

**Discussion by R0Wi** [R0Wi-DEV/workflow_ocr#55] [Ideas]
We'd like to collect some additional configuration options affecting all workflows based on this app.

**Comment by github-cli** [comment]
I think thats a great idea 😄 
language options, overwriting the file on error, other parameters, would all be cool...
an idea: an option to keep the file on errors and a textbox to pass any additional parameters if wanted (and with it replace the --redo-ocr as this would not work with many of the options)
if this is too advanced (for the average end user), simply an option to keep the file and another to select the language
